apps/src/lib/tools/jsdebugger/redux.js

```diff
@@ -282,11 +302,11 @@ export function reducer(state, action) {
     });
   } else if (action.type === APPEND_LOG) {
     return state.merge({
-      logOutput: appendLogOutput(state.logOutput, action.output),
+      logOutput: appendLogOutput(state.logOutput, action.output, action.type),
       maxLogLevel: computeNewMaxLogLevel(state.maxLogLevel, action.logLevel)
     });
   } else if (action.type === CLEAR_LOG) {
-    return state.merge({logOutput: '', maxLogLevel: ''});
+    return state.merge({logOutput: flagOnState(), maxLogLevel: ''});
```

Comment:
Should this change also be hidden behind the experiment flag?

Reply:
Woops! I thought by calling `flagOnState()` which has an experiment flag in it, that it would return `''` when the flag if off and return `[]` when the flag is on. Did I misunderstand? Is `if (experiments.isEnabled('react-inspector')) {state.merge({logOutput: [], maxLogLevel: ''})} else {state.merge({logOutput:'', maxLogLevel: ''})}` a better way to write this?

I also plan on making this into a ternary like so: `experiments.isEnabled('react-inspector') ? [] : ''`. Let me know what you think!